02_activities/assignments/Assignment2.md

```diff
@@ -9,7 +9,7 @@
 * What to submit for this assignment:
     * This markdown (Assignment2.md) with written responses in Section 1
     * Two Entity-Relationship Diagrams (preferably in a pdf, jpeg, png format).
-    * One .sql file 
+    * One .sql file
```

Comment:
Sorry about all the trailing whitespace removals in this PR - I had the setting ON in the editor. Will disable it for the next PRs.